EFCore.VisualBasic/EFCore.VisualBasic.vbproj

```diff
@@ -35,6 +46,10 @@
     <Compile Remove="build\**\*.vb" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="VisualBasicSnapshotGenerator.vb~RF232be437.TMP" />
+  </ItemGroup>
+
   <ItemGroup>
     <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="2.1.0-preview1-*" />
```

Comment:
@sputier I believe we could upgrade to preview2, there may be things in preview1 that no longer exist in preview2.